compiler/rustc_abi/src/lib.rs

```diff
@@ -221,6 +221,19 @@ impl ReprOptions {
 /// * Cranelift stores the base-2 log of the lane count in a 4 bit integer.
 pub const MAX_SIMD_LANES: u64 = 1 << 0xF;
 
+/// Informations relative to a specific address space.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub struct AddressSpaceInfo {
```

Comment:
I said I don't care much about the LLVM naming schemes for things but I'm still confused about this divergence, since I feel this is more vague than what it is
```suggestion
/// How pointers are represented in a given address space
#[derive(Copy, Clone, Debug, PartialEq, Eq)]
pub struct PointerSpec {
```

Reply:
[58055f8](https://github.com/rust-lang/rust/pull/143182/commits/58055f8eafd52dd001eda3a8e5f57d0aa68368df) changes the name.